config/piwik.yml

```diff
@@ -0,0 +1,33 @@
+# Configuration:
+# 
+# disabled
+#   false if tracking tag should be shown
+# use_async
+#   Set to true if you want to use asynchronous tracking
+# url
+#   The url of your piwik instance (e.g. localhost/piwik/
+# id_site
```

Comment:
why are these two things here and in `ENV`? they should be either one place or another but not both, right?